installer/test/InstallerFunctionalTests/constants.h

```diff
@@ -17,29 +17,29 @@
 #endif
 
 #define BASE_PATH L"..\\..\\..\\"
-#define INSTALLER_DIR L"WindowsAppSDKInstall"
-#define INSTALLER_FILENAME L"WindowsAppSDKInstall.exe"
+#define INSTALLER_DIR L"WindowsAppRuntimeInstall"
+#define INSTALLER_FILENAME L"WindowsAppRuntimeInstall.exe"
 #define INSTALLER_EXE_PATH BASE_PATH CONFIGURATION L"\\" ARCH L"\\" INSTALLER_DIR L"\\" INSTALLER_FILENAME
 
-namespace WindowsAppSDKInstallerTests
+namespace WindowsAppRuntimeInstallerTests
 {
     static const int c_phaseTimeout = (30 * 1000); // 30 seconds
 
-    static const std::wstring c_x86FrameworkName = L"Microsoft.WindowsAppSDK.Test.InstallerFramework_1.0.0.0_x86__8wekyb3d8bbwe";
-    static const std::wstring c_x64FrameworkName = L"Microsoft.WindowsAppSDK.Test.InstallerFramework_1.0.0.0_x64__8wekyb3d8bbwe";
-    static const std::wstring c_arm64FrameworkName = L"Microsoft.WindowsAppSDK.Test.InstallerFramework_1.0.0.0_arm64__8wekyb3d8bbwe";
+    static const std::wstring c_x86FrameworkName = L"Microsoft.WindowsAppRuntime.Test.InstallFramework_1.0.0.0_x86__8wekyb3d8bbwe";
+    static const std::wstring c_x64FrameworkName = L"Microsoft.WindowsAppRuntime.Test.InstallFramework_1.0.0.0_x64__8wekyb3d8bbwe";
+    static const std::wstring c_arm64FrameworkName = L"Microsoft.WindowsAppRuntime.Test.InstallFramework_1.0.0.0_arm64__8wekyb3d8bbwe";
 
-    static const std::wstring c_x86MainName = L"WindowsAppSDK.Test.InstallerMain_1.0.0.0_x86__8wekyb3d8bbwe";
-    static const std::wstring c_x64MainName = L"WindowsAppSDK.Test.InstallerMain_1.0.0.0_x64__8wekyb3d8bbwe";
-    static const std::wstring c_arm64MainName = L"WindowsAppSDK.Test.InstallerMain_1.0.0.0_arm64__8wekyb3d8bbwe";
-    static const std::wstring c_mainFamilyName = L"WindowsAppSDK.Test.InstallerMain_8wekyb3d8bbwe";
+    static const std::wstring c_x86MainName = L"WindowsAppRuntime.Test.InstallerMain_1.0.0.0_x86__8wekyb3d8bbwe";
```

Comment:
Future - consider `constexpr` and/or using `std::wstring_view`

Reply:
@dkbennett for future reference

@jonwis string constants are common thing. Is this worth noting in the [C++ coding guidelines](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/Coding-Guidelines/Languages-CPP.md)?

Reply:
this is probably ok for test code. but i'd think plain WCHAR[] is preferred to avoid dynamic initializer in the code and invoked at runtime.